src/pages/api/callAssignments/setCallerTags.ts

```diff
@@ -0,0 +1,99 @@
+import { z } from 'zod';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+import { CallAssignmentCaller } from 'features/callAssignments/apiTypes';
+import { ZetkinTag } from 'utils/types/zetkin';
+import { ApiFetch, createApiFetch } from 'utils/apiFetch';
+
+const bodySchema = z.object({
+  assignmentId: z.number(),
+  callerId: z.number(),
+  excludedTags: z.array(z.number()),
+  orgId: z.number(),
+  prioTags: z.array(z.number()),
+});
```

Comment:
I need to understand what zod does and what it means but it looks interesting at least!